system/forms/formbuilder/item-types/date.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please look at the spacing here (particularly look in github diff view). Use spaces for _alignment_ and never tabs (use tabs just for indentation) :)